pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
+
+- Start a presentation from a markdown file:
+
+`mdp {{presentation.md}}`
+
+- Disable fading transitions:
+
+`mdp --nofade {{presentation.md}}`
+
+- Invert colors for light background:
```

Comment:
**mpd** assumes dark background by default, so `--invert` swaps white and black so it can be used with light backgrounds.  What about change it to just *"Use dark font for light backgrounds"* , or something like that?